Makefile

```
@@ -0,0 +1,99 @@
+include frontend/.env
+include backend/.env
+
+.PHONY: all
+all: test build
+
+.PHONY: run-frontend
+run-frontend: build-frontend
+	cd frontend && npm run dev
+
+.PHONY: run-backend
+run-backend: build-backend
+	cd backend && cargo run
+
+.PHONY: run-mdbook
+run-mdbook: build-mdbook
+	mdbook serve --open
+
+.PHONY: run-storybook
+run-storybook:
+	cd frontend && npm install && npm run storybook
```

Needed according to #439 (probably a sub target `build-frontend-doc` to build frontend doc makes sense?)

```suggestion
	cd frontend && npm install && npm run generate-api-types && npm run doc && npm run storybook
```